tcms/testcases/views.py

```diff
@@ -150,11 +140,25 @@ def post(self, request, *args, **kwargs):
 
         context_data = {
             'test_plan': test_plan,
-            'form': form
+            'form': form,
+            'notify_form': notify_form
         }
 
         return render(request, self.template_name, context_data)
 
+    def _create_test_case(self, form, test_plan):
```

Comment:
Doesn't pylint complain about accessing a private method ? There isn't really a need to rename the method.

Reply:
No, it does not complain. And I don't see why it should since we only use the private from within the class. Actually the naming here is irrelevant, because this is a view class and it is not like someone will do:
```python
view = NewCaseView()
view._create_test_case()
```
so I will revert the name

Reply:
no